.travis.yml

```diff
@@ -100,7 +103,7 @@ script:
   - |
     if [[ $BUILD_DOCS == false ]]; then
       export MPL_REPO_DIR=$PWD  # needed for pep8-conformance test of the examples
-      gdb -return-child-result -batch -ex r -ex bt --args python tests.py -s --processes=$NPROC --process-timeout=300 $TEST_ARGS
+      gdb -return-child-result -batch -ex r -ex bt --args python tests.py $TEST_ARGS
```

Comment:
I think we should do one config with coverage, the rest with multiprocess.  multiprocess catches a certain class of bugs that I'd hate to stop catching.